fga/store/cachedFgaTenantStore.go

```diff
@@ -0,0 +1,96 @@
+package store
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"slices"
+	"time"
+
+	"github.com/hashicorp/golang-lru/v2/expirable"
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"google.golang.org/grpc/status"
+)
+
+type FgaTenantStore interface {
```

Comment:
```suggestion
type FGAStoreHelper interface {
```

I would propose a different naming for the interface

Reply:
done